plugin-gestures/src/test/java/com/mapbox/maps/plugin/gestures/GesturesPluginTest.kt

```diff
@@ -423,6 +423,51 @@ class GesturesPluginTest {
     verify { cameraAnimationsPlugin.easeTo(any(), any()) }
   }
 
+  @Test
+  fun verifyMoveListenerPinchScrollDisabled() {
```

Comment:
Would be nice if add a verifyMoveListenerPinchScrollEnabled

Reply:
The enabled case is the default behaviour and is covered in the above test case. Or do you mean specifically call the `presenter.pinchScrollEnabled = true`?

Reply:
Added a separate test case for setting it to true.